app/main.go

```diff
@@ -2,38 +2,58 @@ package main
 
 import (
 	"flag"
+	"io"
 	"log"
 	"os"
 
 	"github.com/joho/godotenv"
 )
 
 // flags
-// if --Block, use block responses
-var isBlockFormat = flag.Bool("block", false, "Use non-streaming block responses")
+// if --vv, use block responses
+var isVoiceVox = flag.Bool("vv", false, "Use voicevox models")
+var characterVoiceId = flag.String("cid", "13", "Character voice id")
 
 // Env vars
 var openAiApiKey = "OPENAI_API_KEY"
 
 func main() {
 	loadEnv()
 	openaiAPIKey := os.Getenv(openAiApiKey)
+	voiceVoxApiUrl := os.Getenv("VOICEVOX_API_URL")
 	if openaiAPIKey == "" {
 		log.Fatalf("%s not set", openAiApiKey)
 	}
 	flag.Parse()
-	client := CreateChatClient(openaiAPIKey, *isBlockFormat)
-	if *isBlockFormat {
+	chatClient := CreateChatClient(openaiAPIKey, *isVoiceVox)
+	if *isVoiceVox {
```

Comment:
Let's use a guard clause here.
Or actually, you suggested abstracting this code away from main.go.